client/extensions/woocommerce/app/settings/taxes/taxes-wcs.js

```diff
@@ -0,0 +1,237 @@
+import {
+	areTaxSettingsLoaded,
+	getPricesIncludeTax,
+	getShippingIsTaxFree,
+} from 'woocommerce/state/sites/settings/tax/selectors';
+import ExtendedHeader from 'woocommerce/components/extended-header';
+import { updateTaxesEnabledSetting } from 'woocommerce/state/sites/settings/general/actions';
+import QuerySettingsGeneral from 'woocommerce/components/query-settings-general';
+import { fetchTaxRates } from 'woocommerce/state/sites/meta/taxrates/actions';
+import { fetchTaxSettings, updateTaxSettings } from 'woocommerce/state/sites/settings/tax/actions';
+import { getLink } from 'woocommerce/lib/nav-utils';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import Main from 'components/main';
+import TaxSettingsSaveButton from './save-button';
+import SettingsNavigation from '../navigation';
+import { successNotice, errorNotice } from 'state/notices/actions';
+import StoreAddress from 'woocommerce/components/store-address';
+import TaxesOptions from './taxes-options';
+import TaxesRates from './taxes-rates';
+
+class SettingsTaxesWooCommerceServices extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			isSaving: false,
+			pricesIncludeTaxes: props.pricesIncludeTaxes,
+			shippingIsTaxable: props.shippingIsTaxable,
+			taxesEnabled: props.taxesEnabled,
+			userBeganEditing: false,
+		};
+	}
+
+	static propTypes = {
+		site: PropTypes.shape( {
+			slug: PropTypes.string,
+			ID: PropTypes.number,
+		} ),
+		className: PropTypes.string,
+	};
+
+	componentDidMount = () => {
+		const { site } = this.props;
+
+		if ( site && site.ID ) {
+			this.props.fetchTaxSettings( site.ID );
+		}
+	};
+
+	componentWillReceiveProps = newProps => {
+		if ( ! this.state.userBeganEditing ) {
+			const { site } = this.props;
+			const newSiteId = ( newProps.site && newProps.site.ID ) || null;
+			const oldSiteId = ( site && site.ID ) || null;
+			if ( oldSiteId !== newSiteId ) {
+				this.props.fetchTaxSettings( newSiteId );
+			}
+
+			this.setState( {
+				pricesIncludeTaxes: newProps.pricesIncludeTaxes,
+				shippingIsTaxable: newProps.shippingIsTaxable,
+				taxesEnabled: newProps.taxesEnabled,
+			} );
+		}
+	};
+
+	onEnabledChange = () => {
+		this.setState( { taxesEnabled: ! this.state.taxesEnabled, userBeganEditing: true } );
+	};
+
+	onCheckboxChange = event => {
+		const option = event.target.name;
+		const value = event.target.checked;
+		this.setState( { [ option ]: value, userBeganEditing: true } );
+	};
+
+	pageHasChanges = () => {
+		return this.state.userBeganEditing;
+	};
+
+	onSave = ( event, onSuccessExtra ) => {
+		const { site, translate } = this.props;
+
+		event.preventDefault();
+		this.setState( { isSaving: true } );
+
+		const onSuccess = () => {
+			this.setState( { isSaving: false, userBeganEditing: false } );
+			if ( onSuccessExtra ) {
+				onSuccessExtra();
+			}
+			return successNotice( translate( 'Settings updated successfully.' ), {
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Theme settings updated successfully.' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/17110-store-tax-settings-detect-taxjar)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).